### PR TITLE
Build against OpenSearch 1.0.0-alpha1.

### PR DESCRIPTION
**Pull request by dblock**
Signed-off-by: dblock <dblock@amazon.com>

### Description
Build against OpenSearch 1.0.0-alpha1.

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -87,7 +87,7 @@ testClusters.javaRestTest {
 
 allprojects {
     group = 'com.amazon.opendistroforelasticsearch'
-    version = "${opendistroVersion}.0"
+    version = "${opendistroVersion}"
```
Comment (saratvemulapalli) [inline build.gradle:90]: The last digit is intended for plugins to have their own minor version.
Could you help me understand why we'd like to remove it?

**Review by saratvemulapalli — COMMENTED** [review]
Overall the changes look good to me.

Dropped a minor comment.

File: build.gradle (hunk 1 above)
Comment (dblock) [reply build.gradle:90]: Hmm, it was generating an -alpha1.0 build. I can track where it's used incorrectly or we can leave it. WDYT?

Comment (saratvemulapalli) [reply build.gradle:90]: Yeah because `-alpha1` was appended to the `gradle.properties` file. I would prefer to stick with 1.0.0.0 as we have decided.

Comment (saratvemulapalli) [reply build.gradle:90]: Same here as common-utils, lets have the 4th digit for the minor version.

**Review by saratvemulapalli — APPROVED** [review]
Changes look good to me.

Just a minor comment.
Thanks a lot dB!

File: build.gradle (hunk 1 above)
Comment (dblock) [reply build.gradle:90]: reverting

**Pull request merged**